health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/Hypothesis.js

```
@@ -191,111 +198,135 @@ const Hypothesis = ({ category, assumptions: initialAssumptions,setShowToast,all
-          sortFooterChildren={true}
-          onClose={() => {
-            setAssumptionsPopUp(false)
-          }}
-        ></PopUp>}
+              <Button type={"button"} size={"large"} variation={"primary"} label={t("NO")} onClick={handleCancelDelete} />,
+            ]}
+            sortFooterChildren={true}
+            onClose={() => {
+              setShowPopUp(false);
+            }}
+          ></PopUp>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)